featuros-gradle-plugin/build.gradle

```diff
@@ -0,0 +1,23 @@
+project(':featuros-gradle-plugin') {
+    apply plugin: 'groovy'
+    apply plugin: 'maven-publish'
+
+    dependencies {
+        compile gradleApi()
+        compile localGroovy()
+
+        testCompile 'org.spockframework:spock-core:1.0-groovy-2.3'
```

Comment:
Yeah, let's use groovy distribution that comes with gradle, since this plugin doesn't have dependencies on other modules anyway.